doc/filtering.rst

```diff
@@ -92,11 +92,11 @@ This section contains functions to filter values during iteration.
 .. function:: partition(predicate, gen, param, state)
               iterator:partition(predicate)
 
-   :param x: a value to find
-   :returns: {gen1, param1, state1}, {gen2, param2, state2}
+   :returns: iterator, iterator
 
    The function returns two iterators where elements do and do not satisfy the
-   prediucate. Equivalent to:
+   predicate. See note about multireturn in :ref:`Getting started
+   <multi_return>`. Equivalent to:
 
    .. code-block:: lua
```

Comment:
See below:

> Iterators especially returned in tables to work with :func:`zip` and other functions.

And see 'Actual mistakes' section in my comment in the issue. In short: now all iterators are triplets with a table in first item, this comment is just irrelevant now.